### PR TITLE
SMILE-225-Create Practitioner Referring Resource

### PR DESCRIPTION
**Pull request by amakineni**
Create Practitioner Referring Resource on top of message header, facility, patient resources

### CODE REVIEW

File: hl7v2_default.json [hunk 1]
```diff
@@ -385,6 +386,57 @@
...
+                              "object": {
+                                  "coding": {
+                                  "object": {
+                                      "system": {"const": "http://terminology.hl7.org/CodeSystem/v2-0203"},
+                                      "code": {"const": "PRN"}
+                                  }
+                                  }
+                              }
+                              },
+                              "system": {"const": "http://hl7.org/fhir/sid/us-npi"},
```
Comment (aajaddu) [inline hl7v2_default.json:423]: Let's use the system uri for npi that was already created. You can find it in the CSP created specifically for the practitioner npi.

File: hl7v2_default.json [hunk 2]
```diff
@@ -385,6 +386,57 @@
...
+                      }
+                 ]
+              }
+          }
+      },
+      "request": {
+        "object": {
+          "method": {"const": "PUT"},
+          "url": {"const": "Practitioner"},
+          "ifNoneExist": {"const": "identifier=http://hl7.org/fhir/sid/us-npi|9999999999"}
```
Comment (aajaddu) [inline hl7v2_default.json:435]: The business identifier in the request conditional cannot be hardcoded and must match the value defined in the business identifier section above.

**Review by aajaddu — CHANGES_REQUESTED** [review]

File: hl7v2_default.json (hunk 1 above)
Comment (amakineni) [reply hl7v2_default.json:423]: Due to merge conflicts with main, created another PR https://github.com/mednax-it/Omniparser_Schemas/pull/15 with requested changes.

File: hl7v2_default.json (hunk 2 above)
Comment (amakineni) [reply hl7v2_default.json:435]: Due to merge conflicts with main, created another PR https://github.com/mednax-it/Omniparser_Schemas/pull/15 with requested changes.

**Comment by amakineni** [comment]
Due to merge conflicts with main, created another PR https://github.com/mednax-it/Omniparser_Schemas/pull/15 with requested changes.

**Pull request closed without merging**